### DISCUSSION TITLE
Airflow UI

### DISCUSSION THREAD

**Discussion by Rasika-288** [apache/airflow#38136] [Q&A]
We are upgrading our airflow from airflow 1.10 to 2.6.3. We have successfully upgraded till airflow 2.1.0. But after that we are unable to upgrade it. 
Current python version we are using is python 3.9.0 for airflow 2.6.3. The application is up but UI is not running. We are getting the below error in splunk UI - **AttributeError: 'AnonymousUserMixin' object has no attribute 'perms'**.

**Comment by potiuk** [comment]
Impossible to help to solve your problem - to little information - but likely you have some customization you need to adapt - likely on the plugin/UI side. But this is wild guess, you need to understand what customization you implemented and how they interact with Airflow - you and your team are the only people who might have a chance to know it and investigate it
